topics/transcriptomics/tutorials/scrna-intro/slides_ES.html

```diff
@@ -0,0 +1,1262 @@
+---
+layout: tutorial_slides
+logo: "GTN"
+video: true
+title: "Una introducción al análisis de datos scRNA-seq"
+
+subtopic: single-cell
+priority: 1
+level: Intermediate
+video: true
+zenodo_link: ""
+tags:
+  - single-cell
+questions:
+  - ¿Cómo se comparan las muestras?
+  - ¿Cómo son capturadas las células?
+  - ¿Cuál  es la diferencia entre los métodos Bulk RNA-seq y scRNA-seq?
+  - ¿Por qué es importante clusterizar datos?
+objectives:
+  - Entender las dificultades en la secuenciación y en la amplificación del scRNA-seq y cómo superarlas.
+  - Conocer los tipos de variación en un análisis y cómo controlarlos.
+  - Comprender qué es la reducción de dimensiones y cómo se puede realizar.
+  - Familiarizarse con las principales técnicas de clusterización de datos y cuándo usarlas.
+time_estimation: 30m
+key_points:
+  - Los datos scRNA-seq requieren ser pre-procesados antes que un análisis sea llevado a cabo.
+  - Los grupos de células con perfiles similares son comparados contra otros grupos.
+  - Los problemas de detectabilidad requieren una atención especial en todas las etapas.
+  - La clusterización es una etapa completa del análisis.
+follow_up_training:
+  -
+    type: "internal"
+    topic_name: transcriptomics
+    tutorials:
+        - scrna-preprocessing
+lang: es
+translations:
```

Comment:
just add similar `translations` metadata to the English version and should be good :)

Reply:
Thank you! Yes!